scripts/import/dbSNP_v2/update_new_sets.pl

```diff
@@ -295,6 +312,61 @@ sub dump_new_variation_feature {
   
 }
 
+sub get_structure {
+  my $dbhvar = shift;
+
+  my $get_struc_sth = $dbhvar->prepare(qq[ select variation_set_sub, variation_set_super from variation_set_structure]);
+
+  my %parent;
+  $get_struc_sth->execute() ||die;
+  my $dat = $get_struc_sth->fetchall_arrayref();
+  foreach my $l(@{$dat}){
+    $parent{$l->[0]} = $l->[1] ;
+  }
+  return \%parent;
+
+}
+
+sub recalculate {
+  my $input_file = shift;
+  my $output_file = shift;
+  
+  my $parent = get_structure($dbh);
+  my %concat_sets;
+  
+  open FH, "<", "$input_file" or die "Can not open $input_file: $!";
```

Comment:
It would be neater to use the same convention for file handles, preferably $something, and give them names that sound different.